specs/data_structures.md

```diff
@@ -290,6 +290,8 @@ node.v = h(0x00, serialize(d))
 
 The `namespaceID` message field here is the namespace ID of the leaf, which is a [`NAMESPACE_ID_BYTES`](consensus.md#system-parameters)-long byte array.
 
+Leaves in a NMT **must** be lexicographically sorted by namespace ID in ascending order.
```

Comment:
I think it should be "an NMT"

Reply:
Fixed in 19a80df.

Reply:
https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html

NMT is pronounced "en em tee", thus should start be prefixed with "an".